rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/synonyms/20_synonyms_get.yml

```diff
@@ -55,6 +55,38 @@ setup:
         - synonyms: "bye => goodbye"
           id: "test-id-2"
 
+---
```

Comment:
Is it worth adding a test here to test that you can't insert a new synonym after reaching the max?

Reply:
I did that as part of the integration tests - it would be very inconvenient to create 100k docs in a YAML test, and we can use randomization to increase coverage as well